backends/kv/writer.go

```diff
@@ -354,8 +359,13 @@ func (a *Appender) respWaitLoop(timeout time.Duration) {
 			timer.Reset(timeout)
 
 			if resp.Error != nil {
-				a.logger.ErrorWith("failed write response", "error", resp.Error)
-				a.asyncErr = resp.Error
+				// If condition was evaluated as false log this and discard error.
+				if isFalseConditionError(resp.Error) {
+					a.logger.Info("condition was evaluated to false for item %v", resp.Request().Input)
```

Comment:
We're going to log this every time the condition is false?

Reply:
Yes. That's what i get from the ticket.. i am open to changing it to `debug` to not potentially spam the logs but whatever

Reply:
Will leave as info for now.